mono/dodge_the_creeps/Dodge the Creeps with C#.csproj

```
@@ -1,6 +1,12 @@
-<Project Sdk="Godot.NET.Sdk/4.0.0-dev5">
+<Project Sdk="Godot.NET.Sdk/4.0.0">
   <PropertyGroup>
-    <TargetFramework>net472</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
```

`EnableDynamicLoading` is also added to the csproj when creating new projects.

```suggestion
    <TargetFramework>net6.0</TargetFramework>
    <EnableDynamicLoading>true</EnableDynamicLoading>
```